bootstrapper/internal/helm/helm.go

```diff
@@ -215,7 +213,10 @@ type installDoer struct {
 func (i installDoer) Do(ctx context.Context) error {
 	i.log.With(zap.String("chart", i.chart.Name())).Infof("Trying to install Helm chart")
 
-	_, err := i.client.RunWithContext(ctx, i.chart, i.values)
+	if _, err := i.client.RunWithContext(ctx, i.chart, i.values); err != nil {
+		i.log.With(zap.Error(err)).Errorf("Helm chart installation failed")
```

Comment:
Should this log message also have the chart name as tag?

Reply:
Good idea, will add